src/components/tabs/tabs.js

```diff
@@ -237,6 +242,10 @@ export default {
     navWrapperClass: {
       type: [String, Array, Object],
       default: null
+    },
+    keyNav: {
+      type: Boolean,
+      default: false
```

Comment:
Disabling `keyNav` by default will be a BREAKING CHANGE. Also, accessibility enhancements (like keyboard navigation) are enabled in many other areas so is there any reason to disable it by default?

Reply:
That's true. I pushed a fix for this